cmake/DefinePlatformSpecifc.cmake

```diff
@@ -101,7 +101,7 @@ else (CYGWIN)
 	      else()
 	        add_definitions( -DPOCO_HAVE_FD_EPOLL)
 	      endif()
-	      # set(SYSLIBS  pthread ${CMAKE_DL_LIBS} rt)
+	      set(SYSLIBS  pthread ${CMAKE_DL_LIBS} rt)
```

Comment:
It may break compatibility. Will see...

Reply:
I've checked this separately in https://github.com/ClickHouse/ClickHouse/pull/26262 and looks like everything is OK.